packages/react/src/atomic/FileUploader/src/FileUploader.tsx

```
@@ -142,6 +128,35 @@ const FileUploader: React.FC<FileUploaderProps> & FileUploaderComponents = ({
+    return (
+      <div
+        className={fileUploader.classnames.asOverlay}
+        style={assignInlineVars({
+          [vars.width]: width,
+          [vars.height]: height,
+        })}
+        onDragEnter={handleDragEnter}
+        onDragOver={handleDragOver}
+        onDragLeave={handleDragLeave}
+        onDrop={handleDrop}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)